mobile/lib/features/stats/view/stats_view.dart

```diff
@@ -0,0 +1,173 @@
+      measureFn: (s, __) => s.value,
+      data: data,
+      colorFn: (s, __) => charts.ColorUtil.fromDartColor(s.color),
+      labelAccessorFn: (s, __) => '${s.text}: ${s.value}',
+    );
+  }
+
+  charts.Series<_StringIntColorStats, int> generateLivingDeadSeries(
+      BuildContext context) {
+    final livingStats = _StringIntColorStats(
+        "Living", stats.numberOfLivingPeople, Colors.lightGreen);
+    final deadStats = _StringIntColorStats(
+        "Deceased", stats.numberOfDeceasedPeople, Colors.brown);
+
+    return charts.Series<_StringIntColorStats, int>(
+      id: "Living/Dead",
+      domainFn: (s, __) => s == livingStats ? 0 : 1,
+      measureFn: (s, __) => s.value,
+      data: [livingStats, deadStats],
+      colorFn: (s, __) => charts.ColorUtil.fromDartColor(s.color),
+      labelAccessorFn: (s, __) => '${s.text}: ${s.value}',
+    );
+  }
+
+  charts.Series<_StringIntColorStats, int> generateMarriedSingleSeries(
+      BuildContext context) {
+    final marriedStats = _StringIntColorStats(
+        "Married", stats.numberOfMarriedPeople, Colors.lightGreen);
+    final singleStats = _StringIntColorStats(
+        "Single", stats.numberOfSinglePeople, Colors.brown);
+
+    return charts.Series<_StringIntColorStats, int>(
+      id: "Married/Single",
+      domainFn: (s, __) => s == marriedStats ? 0 : 1,
+      measureFn: (s, __) => s.value,
+      data: [marriedStats, singleStats],
+      colorFn: (s, __) => charts.ColorUtil.fromDartColor(s.color),
+      labelAccessorFn: (s, __) => '${s.text}: ${s.value}',
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return SingleChildScrollView(
+        child: Padding(
+      padding: const EdgeInsets.all(16.0),
+      child: Column(
+        children: [
+          Text(
+            "Here are various curiosities related to this family ☺",
+            style: theme.textTheme.headline5,
+            textAlign: TextAlign.center,
+          ),
+          const Divider(),
+          const StatsLabel("Family members total count:"),
+          const SizedBox(height: 8.0),
+          Text(
+            "${stats.totalNumberOfPeople}",
+            style: theme.textTheme.headline1,
+            textAlign: TextAlign.center,
+          ),
+          const Divider(),
+          LabeledPieChart(
+              label: "Living and dead family members counts:",
+              series: generateLivingDeadSeries(context)),
+          if (stats.averageLifespanInDays != 0) ...[
+            const Divider(),
+            const StatsLabel("Family members average lifespan:"),
+            Text(
+              "${(stats.averageLifespanInDays / 365.25).toStringAsFixed(2)} yrs.",
+              style: theme.textTheme.headline2,
+              textAlign: TextAlign.center,
+            ),
+          ],
+          const Divider(),
+          LabeledPieChart(
+              label: "Family members genders counts:",
```

Comment:
As far as I am concerned; "genders count"

Reply:
That's also not true, genders count is always the same in our app - 4.